chapters/data-analysis.tex

```diff
@@ -1,292 +1,487 @@
-so that it is clear how de-identification is done and where it lives.
-Typically, this process only involves stripping fields from raw data,
-naming, formatting, and optimizing the file for storage,
-and placing it as a \texttt{.dta} or other data-format file
-into the de-identified data folder.
-The underlying data structure is unchanged:
-it should contain only fields that were collected in the field,
-without any modifications to the responses collected there.
-This creates a survey-based version of the file that is able
-to be shared among the team without fear of data corruption or exposure.
-Only a core set of team members will have access to the underlying
-raw data necessary to re-generate these datasets,
-since the encrypted raw data will be password-protected.
-The de-identified data will therefore be the underlying source
-for all cleaned and constructed data.
-It will also become the template dataset for final public release.
-
-%------------------------------------------------
+Secure storage of the raw\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Security}}
+data means access to it will be restricted even inside the research team.
+Loading encrypted data multiple times it can be annoying.
+To facilitate the handling of the data, remove any personally identifiable information from the data set.
+This will create a de-identified data set, that can be saved in a non-encrypted folder. 
+De-identification,\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}}
+at this stage, means stripping the data set of direct identifiers such as names, phone numbers, addresses, and geolocations.\sidenote{\url{ https://www.povertyactionlab.org/sites/default/files/resources/J-PAL-guide-to-deidentifying-data.pdf}}
+The resulting de-identified data will be the underlying source for all cleaned and constructed data.
+Because identifying information is typically only used during data collection, 
+to find and confirm the identity of interviewees, 
+de-identification should not affect the usability of the data.
+In fact, most identifying information can be converted into non-identified variables for analysis purposes
+(e.g. GPS coordinates can be translated into distances). 
+However, if sensitive information is strictly needed for analysis,
+all the tasks described in this chapter must be performed in a secure environment.
 
-\section{Cleaning and constructing derivative datasets}
+% Unique ID and data entry corrections ---------------------------------------------
+There are two main cases when the raw data will be modified during data cleaning.
+The first one is when there are duplicated entries in the data.
+Ensuring that observations are uniquely and fully identified\sidenote{\url{https://dimewiki.worldbank.org/wiki/ID\_Variable\_Properties}}
+is possibly the most important step in data cleaning
+(as anyone who ever tried to merge data sets that are not uniquely identified knows).
```

Comment:
Merging a data set that is not fully unique is just a problem if you are doing a 1:1 merge. So this sentence needs to be more specific.

Reply:
How about:

> The first one is when there are improperly duplicated entries in the data.
> Ensuring that observations are correctly and fully identified\sidenote{\url{https://dimewiki.worldbank.org/wiki/ID\_Variable\_Properties}}
> is possibly the most important step in data cleaning.

Reply:
Agreed. All you did was to remove the last sentence "_(as anyone who ever tried to merge data sets that are not uniquely identified knows)_", right?

Reply:
done in d09c32d